style.css

```diff
@@ -439,7 +439,7 @@ span {
 
 .nav-item {
   margin-bottom: 20px;
-  padding-top: 55px;
+  padding-top: 12px;
   padding-left: 14px;
   position: relative;
```

Comment:
- [ ] In my opinion, you should stick to the design as suggested by the previous reviewer by bringing back the separators to the nav items.
- your design:
![your desing](https://user-images.githubusercontent.com/97834160/210368033-6e6a79d5-c892-4f2f-8cba-327b6653819e.png)
- Figma design:
![figma](https://user-images.githubusercontent.com/97834160/210368111-feeb1750-8f0d-44ad-8d0d-6c968d9109b7.png)

Reply:
I guess you are looking at the wrong link. Please check again.

Reply:
Hi @RitaDee, if you have any doubt you might be right, please don't hesitate to ask for a second opinion by using this [link](https://airtable.com/shrQAqnBwek5a0O0s)

Reply:
> I guess you are looking at the wrong link. Please check again.

I see all the changes requested has been implemented